From c13cc14872f0a6a68b7afe2b8e93879f154c42b4 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@turris.com>
Date: Thu, 24 Aug 2023 14:48:43 +0200
Subject: [PATCH] target/linux: Disable MTD_SPLIT_H3C_VFS on mpc85xx

There is a new option MTD_SPLIT_H3C_VFS that needs decision whether to
enable it or not on 5.10 kernel on mpc85xx. As we don't need it, let's
disable it so builds can continue.
---
 target/linux/mpc85xx/config-5.10 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mpc85xx/config-5.10 b/target/linux/mpc85xx/config-5.10
index e815cd34e5..f838081b0f 100644
--- a/target/linux/mpc85xx/config-5.10
+++ b/target/linux/mpc85xx/config-5.10
@@ -334,3 +334,4 @@ CONFIG_BLK_DEV_SD=y
 CONFIG_MTD_NAND_ECC_SW_BCH=y
 # CONFIG_KERNEL_XZ is not set
 CONFIG_KERNEL_GZIP=y
+CONFIG_MTD_SPLIT_H3C_VFS=n
-- 
2.41.0

